src/Microsoft.AspNet.Razor.Runtime/Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -42,7 +43,10 @@ internal TagHelperExecutionContext(string tagName, TagMode tagMode)
         /// <see cref="ITagHelper"/>s</param>
         /// <param name="uniqueId">An identifier unique to the HTML element this context is for.</param>
         /// <param name="executeChildContentAsync">A delegate used to execute the child content asynchronously.</param>
-        /// <param name="startTagHelperWritingScope">A delegate used to start a writing scope in a Razor page.</param>
+        /// <param name="startTagHelperWritingScope">
+        /// A delegate used to start a writing scope in a Razor page and to override the page's current
```

Comment:
Hmm, when mentioning overriding the `HtmlEncoder` on the page it may be worth adding a remarks or additional detail stating that a `null` value results in no value being overridden.